transaction.js

```diff
@@ -10,6 +10,7 @@ var MessageStream = require('./messagestream');
 var trans = exports;
 
 var MAX_HEADER_LINES = config.get('max_header_lines') || 1000;
+var REMOVE_CARRIAGE_RETURN = config.get('remove_carriage_return') || 'true';
```

Comment:
This should be in smtp.ini (where you can define what is a Boolean)

> On Jan 21, 2016, at 6:54 PM, Ricardo Polo <notifications@github.com> wrote:
> 
> In transaction.js:
> 
> > @@ -10,6 +10,7 @@ var MessageStream = require('./messagestream');
> >  var trans = exports;
> >  
> >  var MAX_HEADER_LINES = config.get('max_header_lines') || 1000;
> > +var REMOVE_CARRIAGE_RETURN = config.get('remove_carriage_return') || 'true';
> I don't use a boolean, because I read the false and true from a config file, so it comes as string.
> 
> If I use if (REMOVE_CARRIAGE_RETURN ) it will pass if the string has any content, not just true. Thats why I use if (REMOVE_CARRIAGE_RETURN === 'true')
> 
> So, any adivice about how to convert that value from the config to a boolean? 
> I tried var REMOVE_CARRIAGE_RETURN = Boolean(config.get('remove_carriage_return');)
> with not success, it always returns true.
> 
> I'll add tests
> I'll rename the variable.
> 
> —
> Reply to this email directly or view it on GitHub.
>